src/Language/Pirouette/Example/StdLib.hs

```diff
@@ -0,0 +1,116 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DeriveLift #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE UndecidableInstances #-}
+-- -l has the effect of causing the TH interpreter to try loading
+-- z3 to resolve otherwise undefined symbols
+{-# OPTIONS_GHC -lz3 #-}
```

Comment:
That's interesting! Does this module really depend on z3?

Reply:
I must just have copied this from another file; my bad. 6e6f5f8e937e3254663eadf5a640a779c4c2a7e4 removes it a look just fine.